### PR TITLE
added develeopment guide including one first guideline wrt lua vs. sh…

### PR DESCRIPTION
**Pull request by christf**
based on discussions on IRC there was the idea that it might be useful to collect guidelines for developers. My suggestion is to collect these in this section.

### CODE REVIEW

File: docs/dev/basics.rst [hunk 1]
```diff
@@ -5,6 +5,13 @@ Gluon's source is kept in `git repositories`_ at GitHub.
 
 .. _git repositories: https://github.com/freifunk-gluon
 
+Development Guidelines
+----------------------
+lua should be used instead of sh whenever sensible. The following criteria
+should be considered:
+  - Is the script doing more than just executing external commands? if so, use lua
+  - Is the script parsing/editing json-data? If so, use lua for speed
```
Comment (NeoRaider) [inline docs/dev/basics.rst]: This isn't valid RST, there must be a blank line before the list items, and the list items should not be indented.

**Comment by christf** [comment]
sorry about those syntax errors. This time I ran the rsty ba rst2pdf and id displays correctly.

**Comment by NeoRaider** [comment]
Hmm, I wonder if we really want to add this section to the top of the page, or if it should rather be moved to the bottom.

**Comment by christf** [comment]
this last fixup moves the section to the bottom

**Comment by rubo77** [comment]
Lua is written with a capital "L"

**Comment by rubo77** [comment]
also maybe better english: https://github.com/freifunk-ffm/gluon/pull/1/files

**Pull request merged**